### PR TITLE
Add Support page for Personas

### PR DESCRIPTION
**Pull request by benswannack**
## Context
The MVP of personas made them available on review apps and locally.

## Changes proposed in this pull request
Create personas on QA
Add a page in support that allows users to impersonate the persona users

## Guidance to review
Per commit

## Link to Trello card
https://trello.com/c/IOdsNzSF/4263-make-personas-available-in-qa

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: config/locales/support_interface/personas.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+en:
+  personas:
+    permissions:
+      admin: This user has both manage_organisations and manage_users permissions.
+      user: This user can only view applications.
+    organisation_membership_explanation:
+      self_ratified: This user belongs to a single provider, which has no relationships with other providers.
+      multiple: This user belongs to multiple providers.
+      no_courses: This user belongs to a single provider which has no open courses.
+    users:
```
Comment (benswannack) [inline config/locales/support_interface/personas.yml:10]: I've moved some of the persona user information to the locales. It might be nice to drive the creation service using a separate YML file, but we can do this in another PR if we want to I think

File: lib/tasks/reset_qa.rake [hunk 2]
```diff
@@ -12,6 +12,7 @@ task _reset_qa: %i[
   restore_support_users
   sync_dev_providers_and_open_courses
   generate_test_applications
+  create_persona_users
```
Comment (benswannack) [inline lib/tasks/reset_qa.rake:15]: Is this enough to get them on QA? Will I need to run this manually somehow?

File: lib/tasks/create_persona_users.rake [hunk 3]
```diff
@@ -0,0 +1,5 @@
+desc 'Creates a number of persona provider user accounts'
+task create_persona_users: :environment do
```
Comment (cwrw) [inline lib/tasks/create_persona_users.rake:2]: could you update the docs to include this rake task, as it no longer lives in the local dev setup?

File: lib/tasks/reset_qa.rake (hunk 2 above)
Comment (cwrw) [reply lib/tasks/reset_qa.rake:15]: I'm not sure this runs automatically, I reckon prob best to run `create_persona_users` manually

File: config/locales/support_interface/personas.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+en:
+  personas:
+    permissions:
+      admin: This user has both manage_organisations and manage_users permissions.
```
Comment (cwrw) [inline config/locales/support_interface/personas.yml:4]: maybe we could format the permission names, or maybe put them in `manage_organisations`?

**Review by cwrw — APPROVED** [review]
this is awesome and something we've needed for a while! also love the names

File: config/locales/support_interface/personas.yml (hunk 1 above)
Comment (cwrw) [reply config/locales/support_interface/personas.yml:10]: Yea I think we are currently using the locales to drive the list which isn't ideal in the page, and if we move them to a dedicated file then we can have our source of truth and as you mentioned use that list for creation. But happy for that to come later

Comment (benswannack) [reply config/locales/support_interface/personas.yml:10]: Discussed that we'll raise a card for this and pick it up in a sprint soon
